float32[] joint_goal_list
bool relative
---
bool success